README.md

```diff
@@ -1,33 +1,7 @@
 # EveryBoard
 
-[EveryBoard](https://everyboard.org) is a platform to play various abstract strategy games, but also to develop AI for these games and to explore new games.
+[EveryBoard](https://everyboard.org) is a platform to play various abstract strategy games, to develop AIs for these games, and to explore new game ideas.
```

Comment:
is a plateform to learn and play various abstract strategy games, to battle and develop AIs for these games, and to explore new game ideas. 

?